### PR TITLE
chore/thanks

### PR DESCRIPTION
**Pull request by nothingismagick**
Cleanup Readme

### CODE REVIEW

**Comment by nothingismagick** [comment]
@lemarier - I did not change any code, so have no idea why the tests are suddenly failing. Is it flaky?

**Comment by nothingismagick** [comment]
This is just way out of our control:
```
memory allocation of 16777216 bytes failed
Error: The process 'C:\Rust\.cargo\bin\rustup.exe' failed with exit code 3221226505
```

**Comment by nothingismagick** [comment]
yeah, seems flaky. will let finish, then merge

File: README.md [hunk 1]
```diff
@@ -2,7 +2,15 @@
 
 [![](https://img.shields.io/crates/v/tao?style=flat-square)](https://crates.io/crates/tao) [![](https://img.shields.io/docsrs/tao?style=flat-square)](https://docs.rs/tao/) ![](https://img.shields.io/crates/l/tao?style=flat-square)
 
-Cross-platform application window creation library in Rust that supports all major platforms like Windows, macOS, Linux, iOS and Android.
+[![Chat Server](https://img.shields.io/badge/chat-on%20discord-7289da.svg)](https://discord.gg/SpmNs4S)
+[![devto](https://img.shields.io/badge/blog-dev.to-black.svg)](https://dev.to/tauri)
+[![devto](https://img.shields.io/badge/documentation-tauri.studio-purple.svg)](https://tauri.studio/docs/getting-started/intro)
+[![https://good-labs.github.io/greater-good-affirmation/assets/images/badge.svg](https://good-labs.github.io/greater-good-affirmation/assets/images/badge.svg)](https://good-labs.github.io/greater-good-affirmation)
```
Comment (Beanow) [inline README.md:8]: Image alts don't make a ton of sense here :]

Comment (nothingismagick) [reply README.md:8]: true

**Review by lucasfernog — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (nothingismagick) [reply README.md:8]: but really just an academic thing. i think once we ship out the codebase to the auditors, we will def have time to circle back and make a cohesive revision to all this stuff

**Pull request merged**